docs/guides/admin/docs/modules/player.configuration.md

```diff
@@ -2,32 +2,81 @@ Opencast Player - Configuration
 ===============================
 
 The configurations for the player are done for each tenant. So the configuration keys are located in
-`.../etc/org.opencastproject.organization-mh_default_org.cfg`.
+`.../etc/ui-config/<tenant>/theodul/config.yml`
+
+The default tenant for opencast is `mh_default_org`
 
 Select the Opencast Player
 ------------------------------
 
-To activate the player set:
+To activate the player set in each tenant this line in the file `.../etc/org.opencastproject.organization-<tenant>.cfg`.
+
 
-    prop.player=/engage/theodul/ui/core.html
+    prop.player=/engage/theodul/ui/core.html?id=#{id}
 
 
 Configuration
 -------------
```

Comment:
All the documentation after this line does not really make a lot of sense. Partly, the documentation in the configuration file is already better, in a few cases (e.g. `mastervideotype`) it's not in which case it would make sense to improve the documentation in the configuration file.

It doesn't really make sense to have the same documentation at two places though.

Reply:
I disagree with that because if you are a user, you will not go to each file to read each fragment of the documentation just to look if there is a feature you need.

The documentation in the YAML file is a reminder, but if you need further explanation (Like code examples) you need to go to the documentation.

Reply:
Don't know if I'm atypical, but I did exactly that: go through each configuration file and adjust it to our installation.

Having two places will most definitely result in out of sync documentation. I see the configuration file itself as primary source. The Opencast documentation should give a general overview, but not go over each setting.

Reply:
> Don't know if I'm atypical,

It's not.

Reply:
Deleted the parts that were requested.